src/enarx-wasmldr-tester.rs

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: Apache-2.0
+
+extern crate reqwest;
+#[macro_use]
+extern crate serde_derive;
```

Comment:
This [is not required since](https://doc.rust-lang.org/stable/edition-guide/rust-2018/module-system/path-clarity.html#no-more-extern-crate) `edition = "2018"`.